src/languages/es.ts

```diff
@@ -583,6 +583,9 @@ export default {
             invalidSplit: 'La suma de las partes no equivale al monto total',
             other: 'Error inesperado, por favor inténtalo más tarde',
             genericCreateFailureMessage: 'Error inesperado solicitando dinero, Por favor, inténtalo más tarde',
+            receiptFailureMessage: 'El recibo no se subió. ',
+            saveFileMessage: 'Guarda el archivo ',
+            loseFileMessage: 'o descarta este error y piérdelo',
```

Comment:
NAB: I think it sounds a bit better
```suggestion
            loseFileMessage: 'o descarta este error y pierde el archivo',
```

Reply:
I think using "archivo" twice in a row doesn't quite work: "Guarda el archivo o descarta este error y pierde el archivo" is a bit redundant